cli/waiter/subcommands/tokens.py

```diff
@@ -20,6 +20,7 @@ def print_as_table(query_result):
     """Given a collection of (cluster, token) pairs, formats a table showing the most relevant token fields"""
     cluster_token_pairs = query_result_to_cluster_token_pairs(query_result)
     rows = [collections.OrderedDict([("Cluster", cluster),
+                                     ("Maintenance", token.get('maintenance', False)),
```

Comment:
Can we add a test for this field being displayed?

Reply:
Can we move the maintenance field before `updated` and after `token`.

Reply:
![Screenshot from 2020-11-13 14-53-36](https://user-images.githubusercontent.com/8290559/99120240-22a32d80-25c0-11eb-8e20-73c887bdade1.png)